### PR TITLE
Download PDN with Managed Identity

### PR DESCRIPTION
**Pull request by caaavik-msft**
Will need to make these changes in the 6.0 and 7.0 branches too after this goes in. This is done by downloading PDN in a separate Azure Pipelines build step and the copying it into the helix upload directories during the performance setup scripts.

### CODE REVIEW

**Review by LoopedBard3 — APPROVED** [review]
Looks good presuming a successful test run.

File: scripts/run_performance_job.py [hunk 1]
```diff
@@ -729,18 +729,14 @@ def run_performance_job(args: RunPerformanceJobArgs):
                 os.path.join(args.performance_repo_dir, "src", "tools", "PerfLabGenericEventSourceLTTngProvider", "build.sh"),
                 "-o", os.path.join(payload_dir, "PerfLabGenericEventSourceForwarder")]).run()
         
-        # download PDN
-        if args.os_group == "windows" and args.architecture != "x86" and args.download_pdn:
-            print("Downloading PDN")
-            escaped_upload_token = str(os.environ.get("PerfCommandUploadTokenLinux")).replace("%25", "%")
-            pdn_url = f"https://pvscmdupload.blob.core.windows.net/assets/paint.net.5.0.3.portable.{args.architecture}.zip{escaped_upload_token}"
+        # copy PDN
+        if args.os_group == "windows" and args.architecture != "x86" and args.pdn_path is not None:
```
Comment (DrewScoggins) [inline scripts/run_performance_job.py:733]: Is the reasoning behind this for when people do local runs? We shouldn't need this check as the yaml ensures that pdn_path is empty unless we are in the right OS/Arch combination.

**Comment by DrewScoggins** [comment]
One comment, but LGTM.

**Review by DrewScoggins — APPROVED** [review]

File: scripts/run_performance_job.py (hunk 1 above)
Comment (caaavik-msft) [reply scripts/run_performance_job.py:733]: Yes, the check is just for local runs

**Pull request merged**